rotordc_website_theme/views/website_sale_product_availability_templates.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="products_item_availability" inherit_id="website_sale.products_item">
+        <xpath
+            expr="//div[@itemscope='itemscope']//div[hasclass('product_price')]"
+            position="before"
+        >
+            <div name="product_availability" class="product_availability">
+                <t t-call="website_sale_product_availability.product_availability" />
```

Comment:
I can't find this module : `website_sale_product_availability`

Reply:
https://github.com/coopiteasy/cie-e-commerce/pull/26

It was also in a PR proposed to the OCA, but was never accepted.